### PR TITLE
Final transfer fixes

### PR DESCRIPTION
**Pull request by h-haaks**
#### Pull Request (PR) description
Cleanup README.md
Adjust version_requirements for augeasproviders_core dependency
Remove EoL operatingsystems

### CODE REVIEW

**Comment by h-haaks** [comment]
@bastelfreak I tried to add rhel/centos 9 support but the acceptance tests failed so I decided to drop that.

File: metadata.json [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
   "license": "Apache-2.0",
   "source": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl",
   "project_page": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl",
   "issues_url": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl/issues",
-  "description": "This module provides type/provider for sysctl using the Augeas configuration API library.",
   "dependencies": [
     {
       "name": "puppet/augeasproviders_core",
-      "version_requirement": ">= 2.4.0 < 4.0.0"
+      "version_requirement": ">= 2.4.0 < 5.0.0"
```
Comment (h-haaks) [inline metadata.json:13]: I decided to bump this dependency so that removing EoL operating systems in core is covered.

**Comment by h-haaks** [comment]
Should there be a transfer notice in README.md?

File: metadata.json [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
   "license": "Apache-2.0",
   "source": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl",
   "project_page": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl",
   "issues_url": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl/issues",
-  "description": "This module provides type/provider for sysctl using the Augeas configuration API library.",
   "dependencies": [
     {
       "name": "puppet/augeasproviders_core",
-      "version_requirement": ">= 2.4.0 < 4.0.0"
+      "version_requirement": ">= 3.2.0 < 5.0.0"
```
Comment (h-haaks) [inline metadata.json:13]: 3.2.0 is the first release of puppet/augeasproviders_core
as metioned in https://github.com/voxpupuli/puppet-augeasproviders_sysctl/pull/69/files#r1099799384

Comment (h-haaks) [reply metadata.json:13]: Bump this to 3.2.1 so that the save/load bug workaround is covered?

File: metadata.json [hunk 3]
```diff
@@ -43,37 +42,6 @@
         "7",
         "8"
       ]
-    },
-    {
-      "operatingsystem": "OracleLinux",
```
Comment (bastelfreak) [inline metadata.json:48]: please only purge the operating systems that are EoL. OracleLinux should be fine, I don't know about OpenSUSE 42. FreeBSD 12 and 13 are fine, Arch is fine, OpenBSD 7 is fine as well.

Comment (h-haaks) [reply metadata.json:48]: But how could this module support OSes that are not supported by augeasproviders_core?

File: metadata.json [hunk 4]
```diff
@@ -54,22 +53,20 @@
...
+        "12",
+        "13"
       ]
     },
     {
       "operatingsystem": "OpenBSD",
       "operatingsystemrelease": [
-        "5",
-        "6"
+        "7"
```
Comment (h-haaks) [inline metadata.json:69]: https://endoflife.date/openbsd

File: metadata.json [hunk 5]
```diff
@@ -54,22 +53,20 @@
     {
       "operatingsystem": "OpenSUSE",
       "operatingsystemrelease": [
-        "42.2"
+        "15.4"
```
[the remark this answers is not in the document]
Comment (h-haaks) [reply metadata.json:56]: There is no acceptance testing for OpenSUSE.
Still keep this as supported?

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**